### ISSUE TITLE
Isolate the code of cname management and add it to the changes we did to the library to have it included for the new version correct 

### ISSUE THREAD

**Issue by akramsarkis** [Geeks-Solutions/nuxt-sections#56]
- [ ] Isolate the code of cname management and add it to the changes we did to the library to have it included for the new version correct 

### **Acceptance testing**

- [ ] Make sure cname is working fine as well as all the functionalities of the library

**Comment by mahfouzhseineweev** [comment]
@akramsarkis To test this on sections backend prod, you simply replace the project id in the APIs by the website domain

**Comment by akramsarkis** [comment]
> @akramsarkis To test this on sections backend prod, you simply replace the project id in the APIs by the website domain

@mahfouzhseineweev  I saw your comment for cname, I tried with Maiori website for example the endpoint to get pages, and instead of the project ID, I used the domain ( I updated it in bo section to be the one of the site not localhost ) , it returned a 200 but with an empty list of page

is this because it needs a specific configuration to return indeed the pages ? and the fact that it returns a 200 means that the backend is deployed on prod ?

For get section type it is working fine, it returns the section types listing

**Comment by mahfouzhseineweev** [comment]
@akramsarkis The test you will be performing is on the meta sections project we have that is using sections dev.
For the acceptance testing, you need to cover all the features that were added recently to the library and used on Maiori website such as the QS management, global section types including all its features.

And of course a sanity test on Sections add edit delete section types / add edit delete page / edit page metadata etc...

So in short both the cname update and the last version of nuxt sections 1.0.11 must be available in this update

**Comment by akramsarkis** [comment]
@mahfouzhseineweev 

Here the steps I followed

1) I created a new project on section dev
2) In project info i added for domain project1.saas.k8s-dev.geeks.solutions.com so including k8s-dev.geeks.solutions.com sub domain used by meta section
3) For the magic url I added https://project1.saas.k8s-dev.geeks.solutions.com
4) I saved the project info successfully
5) I tried to access the site clicking on Connect to my site and got the below

<img width="1653" alt="Screenshot 2024-10-25 at 14 14 31" src="https://github.com/user-attachments/assets/d420a23d-cf16-4a86-b5e0-a875edffacc9">

Am I doing something wrong ?

**Comment by akramsarkis** [comment]
1) I created 2 projects that has as sub domain the one of the cname then for each I added a magic url including the domain and accessed the site  and was able to create and edit pages and save them

2) However when accessing the settings and clicked on the media component to add a faveicon  I saw that like for geeks website the media navigator is broken and needs to be fixed

<img width="1621" alt="Screenshot 2024-10-25 at 14 26 13" src="https://github.com/user-attachments/assets/5cc577b5-43b5-4cf2-b600-83eb65e6b6e6">

3)You need to fix the management of the options for the sections like you did in geeks website as now I cannot save page, so this should work automatically when using cname site

**Comment by mahfouzhseineweev** [comment]
> You need to fix the management of the options for the sections like you did in geeks website as now I cannot save page, so this should work automatically when using cname site

@akramsarkis We can not put this point as a QA here as it is not related.
Let us move this point to the issue related to the sections that should be publicly available for meta sections as most of the current ones will be removed if I am not wrong

**Comment by mahfouzhseineweev** [comment]
> I created 2 projects that has as sub domain the one of the cname then for each I added a magic url including the domain and accessed the site and was able to create and edit pages and save them
However when accessing the settings and clicked on the media component to add a faveicon I saw that like for geeks website the media navigator is broken and needs to be fixed

What are the 2 projects you created ? @akramsarkis 
And for media component problem is it also reproducible on the meta sections dev site for all medias and the favicon one ? http://sectionswebsite-saas.k8s-dev.geeks.solutions/

**Comment by akramsarkis** [comment]
As requested the second point was moved to the CSS issue https://github.com/Geeks-Solutions/nuxt-sections/issues/69 and the media navigator is now showing correctly, so closing the cname issue
